src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="center-block">
+    <div class="calendar calendar-first-day-sun calendar--month-length-31">
+      <div class="calendar__day"></div>
```

Comment:
Would be better to do it with list tags
```suggestion
    <ul class="calendar calendar-first-day-sun calendar--month-length-31">
      <li class="calendar__day"></div>
```